synapse/cortex.py

```diff
@@ -3229,6 +3230,32 @@ async def itemsStormVar(self):
         for item in self.stormvars.items():
             yield item
 
+    # Storm package vars APIs
+
+    def _getStormPkgVarKV(self, name):
+        if (pkgvars := self.stormpkgvars.get(name)) is None:
+            self.stormpkgvars[name] = pkgvars = self.cortexdata.getSubKeyVal(f'stormpkg:vars:{name}:')
+        return pkgvars
+
+    async def getStormPkgVar(self, name, key, default=None):
+        pkgvars = self._getStormPkgVarKV(name)
+        return pkgvars.get(key, defv=default)
+
+    @s_nexus.Pusher.onPushAuto('stormpkgvar:pop')
+    async def popStormPkgVar(self, name, key, default=None):
+        pkgvars = self._getStormPkgVarKV(name)
+        return pkgvars.pop(key, defv=default)
+
```

Comment:
exposing `truncate` on the subkeyval would probably be nice to have for clearing everything when uninstalling a package

Reply:
I guess its a bit of a design intent of whether or not the `pkg:del` nexus handler would also cleanup the pkg vars? We definitely could that there.

Reply:
I think we can expose the API to allow for cleanup, but IMO it is out-of-scope to actually implement cleanup since what ``pkg:del`` would do has some interplay with the intent of the TBD "uninstall" handler.